app/cronjob/trailingTrade/step/remove-last-buy-price.js

```diff
@@ -169,15 +171,30 @@ const execute = async (logger, rawData) => {
     return data;
   }
 
-  if (baseAssetQuantity * currentPrice < parseFloat(minNotional)) {
+/**
+ * Get symbol configuration
+ *
+ * @param {*} logger
+ * @param {*} rawData
+ */
+ 
+ //Get symbol config
+  const symbolConfiguration = await getConfiguration(logger, symbol);
```

Comment:
You do not need to `getConfiguration` again because it has been already added to `rawData`.

Refer 
- https://github.com/chrisleekr/binance-trading-bot/blob/master/app/cronjob/trailingTrade/step/get-symbol-configuration.js#L14
- https://github.com/chrisleekr/binance-trading-bot/blob/master/app/cronjob/trailingTrade/step/place-buy-order.js#L31

Reply:
Noted! :)